[2022-04-02 16:13:17,191] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available 2022-04-02T16:11:41.268561+00:00 [queued]>
[2022-04-02 16:13:17,200] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available 2022-04-02T16:11:41.268561+00:00 [queued]>
[2022-04-02 16:13:17,201] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-02 16:13:17,201] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-02 16:13:17,201] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-02 16:13:17,223] {taskinstance.py:1087} INFO - Executing <Task(HttpSensor): is_api_available> on 2022-04-02T16:11:41.268561+00:00
[2022-04-02 16:13:17,225] {standard_task_runner.py:52} INFO - Started process 2692 to run task
[2022-04-02 16:13:17,235] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'is_api_available', '2022-04-02T16:11:41.268561+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpxqp8pz4g', '--error-file', '/tmp/tmpd91tb2es']
[2022-04-02 16:13:17,236] {standard_task_runner.py:77} INFO - Job 28: Subtask is_api_available
[2022-04-02 16:13:17,389] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.is_api_available 2022-04-02T16:11:41.268561+00:00 [running]> on host airflowvm
[2022-04-02 16:13:17,580] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T16:11:41.268561+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-02T16:11:41.268561+00:00
[2022-04-02 16:13:17,581] {http.py:101} INFO - Poking: api/v2/admin/location/states/
[2022-04-02 16:13:17,586] {base.py:69} INFO - Using connection to: id: user_api. Host: https://cdn-api.co-vin.in, Port: None, Schema: , Login: , Password: None, extra: {}
[2022-04-02 16:13:17,587] {http.py:140} INFO - Sending 'GET' to url: https://cdn-api.co-vin.in/api/v2/admin/location/states/
[2022-04-02 16:13:17,847] {base.py:243} INFO - Success criteria met. Exiting.
[2022-04-02 16:13:17,855] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20220402T161141, start_date=20220402T161317, end_date=20220402T161317
[2022-04-02 16:13:17,925] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-04-02 16:13:17,935] {local_task_job.py:151} INFO - Task exited with return code 0
